py/visdom/__init__.py

```diff
@@ -2209,3 +2209,75 @@ def mesh(self, X, Y=None, win=None, env=None, opts=None):
             'layout': _opts2layout(opts),
             'opts': opts,
         })
+    @pytorch_wrap
+    def double_yaxis_lines(self,X=None,Y1=None,Y2=None,opts=None,win=None,env=None):
+        '''
+        This function will create a line plot using plotly with different Y-Axis.
+        `X`  = A numpy array of the range.
+        `Y1` = A numpy array of the same count as X.
+        `Y2` = A numpy array of the same count as X.
+        Here are the `opts` options:
```

Comment:
Nit: `The following `opts` are supported:`. Try to match the style of the other docblocks if possible.

Reply:
Changes Done!